### PR TITLE
test(Cypress): add cellView test

### PR DESCRIPTION
**Pull request by Dfldr**
This PR adds a test that ensures functionality of cell views in dashboards. This PR is a repaired version of PR 3607.

### CODE REVIEW

**Comment by Dfldr** [comment]
@subirjolly Please review my PR.

File: cypress/e2e/shared/dashboardsView.test.ts [hunk 1]
```diff
@@ -1324,4 +1324,158 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
...
+    cy.getByTestID('page-header').should('be.visible')
+    cy.getByTestID('selector-list wumpus').click()
+    cy.getByTestID('selector-list dur').click()
+    cy.getByTestID('time-machine-submit-button').click()
+    cy.getByTestID('save-cell--button').click()
+    cy.wait('@loadQuery')
+    cy.getByTestID('giraffe-layer-line').should('be.visible')
+
+    // band
+    cy.getByTestID('cell-context--toggle').click()
```
Comment (subirjolly) [inline cypress/e2e/shared/dashboardsView.test.ts:1370]: Parameterization is a good option in this case. I'd not recommend parameterizing the entire test.. Instead just create an array, loop over it and put these assertions inside the loop.

**Review by subirjolly — CHANGES_REQUESTED** [review]

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3741)

File: cypress/e2e/shared/dashboardsView.test.ts (hunk 1 above)
Comment (Dfldr) [reply cypress/e2e/shared/dashboardsView.test.ts]: I pushed a commit with requested changes

File: cypress/e2e/shared/dashboardsView.test.ts [hunk 2]
```diff
@@ -1399,4 +1399,71 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
...
+    cy.getByTestID('cell-context--configure').click()
+    cy.getByTestID('page-header').should('be.visible')
+    cy.getByTestID('selector-list wumpus').click()
+    cy.getByTestID('selector-list dur').click()
+    cy.getByTestID('time-machine-submit-button').click()
+    cy.getByTestID('save-cell--button').click()
+    cy.wait('@loadQuery')
+    cy.getByTestID('giraffe-layer-line').should('be.visible')
+    ;(function() {
+      for (let i: number = 0; i < dropdowns.length; i++) {
```
Comment (subirjolly) [inline cypress/e2e/shared/dashboardsView.test.ts:1455]: If you want to go with this then it's fine. But, let's try to use `Array.forEach` for the `for` loops. It's much cleaner as well.

**Pull request merged**